core/mps/StaticCameraCalibrationReader.cpp

```diff
@@ -90,7 +90,7 @@ StaticCameraCalibrations readStaticCameraCalibrations(const std::string& fileNam
     pose.intrinsics = intrinsics;
 
     XR_CHECK(
-        (start_frame_idx == -1 && start_frame_idx == -1) ||
+        (start_frame_idx == -1 && end_frame_idx == -1) ||
```

Comment:
I would just remove this line since the check below is doing the positive checking.
Can you update your diff (by just removing this entire line)?